### PR TITLE
Consolidate structure of Documentation 

### LINKED ISSUE #27: documentation updates: intro to docs and P-a-a-S document move
I think it would be more clear to our users if we:
* had the P-a-a-S document included within the docs/ tree (redirect the former URL if possible)
* and have a intro to the documention available at /docs/ - right now there is just a list of pages

### LINKED ISSUE #38: redirect "packit.dev/about" to packit.dev
https://www.redhat.com/en/blog/never-forget-where-you-come-expanding-and-evolving-red-hat-enterprise-linux-community-ecosystem

the link is 404, not nice

### LINKED ISSUE #93: merge testing farm document with packit as a service doc

### PR DESCRIPTION
**Pull request by jpopelka**
The main intent is to have one structure for the whole documentation
instead of having documents outside the main documentation
(e.g. testing farm, packit-as-a-service).

Fixes #27
Fixes #38
Fixes #93

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # pre-commit install -t pre-commit -t pre-push
 
 repos:
-  - repo: https://github.com/prettier/pre-commit
-    rev: 57f39166b5a5a504d6808b87ab98d41ebf095b46
+  - repo: https://github.com/pre-commit/mirrors-prettier
```
Comment (csomh) [inline .pre-commit-config.yaml:6]: Please add a line in the commit message to explain why this change was done.

File: content/docs/_index.md [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
-* [Architecture](/docs/architecture/)
-* [Configuration for packit]({{< ref "/docs/configuration.md" >}})
-* [Packit CLI](/docs/cli/)
-* [Packit Guide](/docs/guide/)
+* [Frequently Asked Questions]({{< ref "faq" >}}) - Whys, Whats, contacts.
+* [Source-git](/docs/source-git/) - Source-git concept and how to work with it.
+* [Onboarding Guide]({{< ref "guide" >}}) - How to start using Packit.
+* [Configuration for packit]({{< ref "configuration" >}}) - Configuration file description.
+* [Testing Farm]({{< ref "testing-farm" >}}) - Guide to Packit's service for running tests.
+* [Packit-as-a-Service]({{< ref "packit-as-a-service" >}}) - GitHub App that helps you continuously ensure that your project work in Fedora OS.
```
Comment (TomasTomecek) [inline content/docs/_index.md:12]: I'd probably reorder this a bit to:
* [Onboarding Guide]
* [Packit-as-a-Service]
* [Testing Farm]
* [Frequently Asked Questions]
* [Source-git]
* [Configuration for packit]

**Review by TomasTomecek — COMMENTED** [review]
otherwise lgtm, nicely done!

File: layouts/partials/logo.html [hunk 3]
```diff
@@ -1 +1 @@
-<a href="/"><img src="/packit_logo.png"></a>
+<a href="/"><img src="/logo-white.png"></a>
```
[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply layouts/partials/logo.html:1]: Guidelines are [here](https://github.com/packit/packit/blob/master/design/export/logo-guideline.pdf) if you are interested.

[the remark this answers is not in the document]
Comment (jpopelka) [reply layouts/partials/logo.html:1]: yeah, I would if I knew where

[the remark this answers is not in the document]
Comment (csomh) [reply layouts/partials/logo.html:1]: https://learn.netlify.app/en/basics/style-customization/#yours-variant

[the remark this answers is not in the document]
Comment (jpopelka) [reply layouts/partials/logo.html]: Looks like this now:
Screenshot from 2020-12-01 16-38-51

It's based on the blue variant of the theme with only logo background color changed.

**Comment by jpopelka** [comment]
@TomasTomecek @csomh PTAL
